### PR TITLE
Added support for the 'search' endpoint

### PR DESCRIPTION
**Pull request by simpandev**
I added the method GrafanaService.search to support calls to the 'search' endpoint

### CODE REVIEW

File: src/main/java/com/appnexus/grafana/client/models/GrafanaSearch.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/* Licensed under Apache-2.0 */
+package com.appnexus.grafana.client.models;
+
+import java.util.List;
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+@Data
+@Accessors(fluent = true)
+public class GrafanaSearch {
```
Comment (skoka-anx) [inline src/main/java/com/appnexus/grafana/client/models/GrafanaSearch.java]: maybe name this `GrafanaSearchResult`

**Review by skoka-anx — COMMENTED** [review]
Looks good overall. Just a minor comment about the naming of the model.

**Comment by skoka-anx** [comment]
can you fix the formatting? I can merge the change once its fixed and the build passes.

**Comment by bascarsija** [comment]
this seems pretty great -- any chance this will warrant a new 1.0.4 release and a bump to the maven artifact version?

**Pull request merged**